examples/sdk/reactNative/README.md

```diff
@@ -10,6 +10,11 @@ This example app shows features available in the @backtrace/react-native package
 
 #### Source maps
 
+Before executing any step:
+
+-   Please update .backtracejsrc file with your symbols submission URL and your sourcemap settings.
+-   please install @backtrace/javascript-cli library.
```

Comment:
AFAIK the user would have to install this package globally for line 56 to work. Could we use `npx` instead?

Reply:
Good idea!